guides/v2.1/extension-dev-guide/build/create_component.md

```diff
@@ -12,48 +9,46 @@ redirect_from:
   - /guides/v2.0/extension-dev-guide/create_component.html
 ---
 
-## Overview of naming a component {#overview-naming}
 You give a name to your component in its `composer.json` and `module.xml` files. These files also contain other required configuration parameters, such as the module's schema version.
 
 ## Prerequisites {#prereq}
 Before you continue, make sure you have completed all of the following tasks:
-
-*   Created a [file structure]({{ page.baseurl }}/extension-dev-guide/build/module-file-structure.html)
-*   Created the the [configuration files]({{ page.baseurl }}/extension-dev-guide/build/required-configuration-files.html) you'll need
-*   [Registered]({{ page.baseurl }}/extension-dev-guide/build/component-registration.html) your component
+.
+*   Create a [file structure]({{page.baseurl}}/extension-dev-guide/build/module-file-structure.html).
+*   Create the the [configuration files]({{page.baseurl}}/extension-dev-guide/build/required-configuration-files.html) you'll need.
+*   [Register]({{page.baseurl}}/extension-dev-guide/build/component-registration.html) your component.
 
 ## Add the component's `module.xml` file {#module-xml}
-Declare the component itself by adding a {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %}.xml file in the `/etc` folder of your component.
+Declare the component itself by adding a {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}`module`{% endglossarytooltip %}.xml file in the `/etc` folder of your component.
```

Comment:
make sure that `module.xml` (including the .xml) is enclosed in the ticks.

Reply:
👍